packages/cli/src/commands/governance/executehotfix.ts

```diff
@@ -23,7 +27,13 @@ export default class ExecuteHotfix extends BaseCommand {
   async run() {
     const res = this.parse(ExecuteHotfix)
     const account = res.flags.from
-    const hotfix = await buildProposalFromJsonFile(this.kit, res.flags.jsonTransactions)
+
+    const jsonString = readFileSync(res.flags.jsonTransactions).toString()
+    const jsonTransactions: ProposalTransactionJSON[] = JSON.parse(jsonString)
+
+    const builder = new ProposalBuilder(this.kit)
+    jsonTransactions.forEach((tx) => builder.addJsonTx(tx))
```

Comment:
Nit: Prefer `for` loops instead of `.forEach`
```suggestion
    for (const tx of jsonTransactions) {
      builder.addJsonTx(tx)
    }
```